Sources/swift-parser-test/DiagnosticsFormatter.swift

```diff
@@ -0,0 +1,133 @@
+//===------------ DiagnosticsFormatter.swift - Formatter for diagnostics ----------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+
+import Foundation
```

Comment:
Do we actually need `Foundation` here? I'd prefer if we could avoid it, because I think this formatter should be part of the `SwiftDiagnostics` module (which can't import `Foundation`).